contents/blog/mykonos-hackathon.md

```diff
@@ -0,0 +1,153 @@
+
+### HERMES
+
+<BuiltBy people={["Annika Schmid", "Simon Fisher", "Mine Kansu"]} />
+
+At PostHog, we love speaking to our users. Maybe a bit too much. Right now, our master customer interviews doc is 382 pages long and contains almost 200 user interviews. It’s a great resource, but it’s getting a bit unwieldy. But now we’ve entered a bold new era of feedback management at PostHog thanks to HERMES, or, snappily, Holistic Evaluation Repository for Managing Enhancements and Suggestions.
+
+This is effectively a database of user interviews, showing who was interviewed, who they work for, what they do, how much they pay for PostHog, the products they talked to us about, and an AI-generated summary of our user interview notes. The database is searchable and you can easily add new interviews in a couple of clicks.
+
+The database is linked right into PostHog, making it easy to see associated user and organization profiles. It’s also hooked into Vitaly, our customer success tool, to automatically pull in more customer and business information.
+
+HERMES also uses ChatGPT to generate a summary of the features requested during the interview based on the human-authored interview notes. This makes it easy to share actionable feedback from users directly with the product team responsible for that feature. 
+
+As part of the project, we also revamped our system for categorizing and tracking feature requests from customers, making it easier for us to see what people want and prioritize the most important product work.
+
+### Data crunching
+
+<BuiltBy people={["Sandy Spicer", "Tim Glaser"]} />
+
+PostHog crunches a lot of data – and sometimes queries can take a while to load. To help users better understand the hard work we’re doing when they make a query, we built a loading bar that shows statistics on the data we’re crunching, and the CPU we’re throwing at each query. The loading bar is fake right now (sorry!) but the data we show is very much real.
+
+
+### VIP customer lookup
+
+<BuiltBy people={["Simon Fisher"]} />
+
+When a customer reports a problem, we want our engineers to get it fixed as soon as possible – but right now our engineers lack context on which customers to prioritize. Right now, they ping our customer success team on Slack for details on accounts. This isn’t good practice in an [async company](/newsletter/how-we-work-async), especially when this information is already written down somewhere.
+
+Enter the VIP Customer Lookup bot. Ping this clever little Slackbot with any customer ID and it’ll return key details of any account, including the plan they are on, how much money they spend, and who on customer success looks after the account.
```

Comment:
Useful though this for us, I'm not sure we should include this. Not super relevant to readers given it's an entirely internal tool.

Reply:
Yup, I think it's for the best.